### ISSUE TITLE
[REQUEST] Linux Support

### ISSUE THREAD

**Issue by stpnwf** [PixiEditor/PixiEditor#610]
I just found out about this software and it looks like it is everything I am looking for. But sadly it is Windows only and does not support Linux, so I can't try it out =/

So I would like to request that it, eventually, be released for Linux.
If you do, please consider doing so on [Flathub](https://docs.flathub.org/docs/for-app-authors/why-flathub/)

**Comment by flabbet** [comment]
Planned for update 2.0

**Comment by Equbuxu** [comment]
why close, it's not completed yet, and flathub sounds like a good suggestion

**Comment by stpnwf** [comment]
> This is planned for update 2.0

Awesome. If I may ask, by any chance, are there any estimates on when 2.0 will be released? Just so I can curb my expectations, is it still a long way off or perhaps just around the corner?

**Comment by flabbet** [comment]
Full release will probably not be available this year, however we expect to release development releases in the next months, so if you feel like using a little less stable version, then stay tuned

**Comment by stpnwf** [comment]
Yeah, I'd be down to beta test it and report any issues. Will check the Discord dev-tests channel periodically for updates on it. Thanks

**Comment by flabbet** [comment]
Linux builds are available https://github.com/PixiEditor/PixiEditor-development-channel/releases
At the moment tar.gz with binaries and .deb package is generated
